.github/workflows/docusaurus.yml

```
@@ -0,0 +1,53 @@
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'website/**'
+  workflow_dispatch: # Allow manual triggering
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)